### ISSUE TITLE
[conan center] advice set_property rather set_target_properties

### ISSUE THREAD

**Issue by SSE4** [conan-io/hooks#181]
from https://github.com/conan-io/conan-center-index/pull/1254#discussion_r403488929
/cc @uilianries 
for `CMakeLists.txt` and `test_package/CMakeLists.txt`, we may advice  `set_property` over `set_target_properties` for advanced CMake compatibility

**Comment by Croydon** [comment]
If you are going to implement this, then I would say this should be just a warning, not an error

**Comment by uilianries** [comment]
Yes, also, `set_target_properties` is okay when the required version is CMake 3+ (and really need Cmake 3)
